data/countries/PK.yaml

```diff
@@ -0,0 +1,46 @@
+holidays:
+  PK:
+    names:
+      en: Pakistan
+    langs: []
+    zones:
+      - Asia/Karachi
+    dayoff: sunday
+    days:
+      02-05:
+        type: observance
+        name:
+          en: Kashmir Solidarity Day
+      03-23:
+        type: observance
```

Comment:
Can you pls. check if "type" is correct.
See https://github.com/commenthol/date-holidays/blob/master/docs/specification.md#types-of-holidays

observance means that there that some people celebrate the day
public instead means there is some kind of law which declares the day as public holiday